requirements/dev.txt

```diff
@@ -3,9 +3,9 @@ mock >= 1.0.1
 nose >= 1.3.0
 
 # For documentation.
-Sphinx >= 1.2.1
+Sphinx >= 1.2.1, < 1.5
```

Comment:
I couldn't get the docs to build but I was confused that the PR was green. I discovered that tox isn't triggering the `html` builder because of a typo (you can't specify `-b` more than once). Oops. Can you add this to your patch? It's working for me after this. I could put it on master but it will create a conflict.

```diff
diff --git a/requirements/dev.txt b/requirements/dev.txt
index 674106b..b34c439 100644
--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -3,6 +3,7 @@ mock >= 1.0.1
 nose >= 1.3.0
 
 # For documentation.
+docutils < 0.13.1
 Sphinx >= 1.2.1, < 1.5
 sphinx-rtd-theme >= 0.1.5
 
diff --git a/tox.ini b/tox.ini
index 7a40af1..d018a59 100644
--- a/tox.ini
+++ b/tox.ini
@@ -18,4 +18,5 @@ basepython=python2.7
 changedir=docs
 deps={[base]deps}
 commands=
-    sphinx-build -b html -b doctest -d {envtmpdir}/doctrees .  {envtmpdir}/html
+    sphinx-build -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
+    sphinx-build -b doctest -d {envtmpdir}/doctrees .  {envtmpdir}/doctest

```